src/shed/_cli.py

```diff
@@ -45,13 +44,17 @@ def _guess_first_party_modules(cwd: Optional[str] = None) -> FrozenSet[str]:
     return frozenset(
         p
         for p in {Path(base).name} | provides
-        if p.isidentifier() and place_module(p) != "STDLIB"
+        # TODO: isort.place_module is horrendously complicated, but we only use
+        # a fraction of the functionality. So best approach, if we still need
+        # the ability to exclude stdlib modules here, is probably to generate a list of
+        # known stdlib modules - either dynamically or store in a file.
+        if p.isidentifier()  # and place_module(p) != "STDLIB"
```

Comment:
Yeah, having a known list seems fine, and we can just take the union of stdlib modules in all non-EOL Python versions.

Reply:
or well, depends on how thorough we want to be + the level of support for running-version-being-different-from-checked-version

Reply:
https://pypi.org/project/stdlibs/ sounds pretty great if we want to be maximum thorough

Reply:
Let's just use that on 3.10+, and vendor the 3.10 list for older versions 👍